.NET/Microsoft.Recognizers.Text.DateTime/Utilities/TimeFunctions.cs

```diff
@@ -108,10 +113,6 @@ public DateTimeResolutionResult PackTimeResult(DateTimeExtra<TimeType> extra, Ti
             }
 
             dateTimeResult.Timex = build.ToString();
-            if (hour == 24)

```

Comment:
Why is this no longer needed or causing side-effects?

Reply:
this is now done at line 79 (moved it there because was needed in the if clauses above)